data-model/src/main/avro/RawEventLog.avdl

```diff
@@ -0,0 +1,19 @@
+@namespace("org.hypertrace.core.datamodel")
+protocol LogEventRecordsProtocol {
+  import idl "Attributes.avdl";
+
+  record LogEventRecord {
+    string customer_id;
```

Comment:
```suggestion
    string tenant_id;
```

Reply:
I see raw span uses customer_id, I think it is a good time to change it but I'd spend time adding a comment on what is this and what it represents. Also, a side comment linking this to the RawSpan field.

Reply:
Added a comment